corona-warn-app/cwa-app-android
# Fix app crash when closing cert details after notification (EXPOSUREAPP-14542)

[Ticket.](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14542)

How to test:

- Scan a certificate: `cwa vc gen -e mock-cloud`
- Revoke the signature: `cwa dcc rl -e mock-cloud --data -i` -> paste the barcode data of the certificate you just scanned, hit enter and select "Revoke Signature" and hit enter again.
- In the app, go to the test menu -> `DCC State Validation`, and refresh the revocation list.
- When you get a notification, tap on it to go to the certificate details screen. 
- Close the screen and return to the certificates screen. No crash should take place anymore.
